src/test/java/uk/gov/justice/laa/crime/cfecrime/cma/StubCmaTest.java

```diff
@@ -13,6 +13,7 @@ class StubCmaTest {
     @Test
     void callCma() {
         var result = new StubCma().callCma(null);
+        MapResponse mapResponse = new MapResponse();
```

Comment:
bit confused about this line - why has it been added in this changeset?

Reply:
I have had to add this to create the object to cover the class being used, otherwise the code coverage says 37%.